# motif-discovery
Final project in UCLA graduate course, Bioinformatics 225 - Computational Methods in Genomics.

Exploration of problem statement provided by the DREAM Gene Expression Prediction Challenge, where we were asked to predict gene expression levels from promoter sequences in S. cerevisiae: https://www.synapse.org/#!Synapse:syn2820426/wiki/71010. 

My approach was to use the following features: (1) Log-odds of finding binding motifs for known S. cerevisiae and putative RP TF binding sites, (2) measures of CpG islands, (3) GC content, and (4) free energy in both regression models for the prediction of gene expression levels and classification models for the prediction of gene expression ranges (See report for details on analysis and findings).
